src/components/PageHeader/PageHeader.tsx

```diff
@@ -7,8 +7,9 @@ import Search from 'components/Search/Search'
 
 const PageHeader = ({
   children,
+  searchDisplay = true,
 }: {
-  searchQuery?: string
+  searchDisplay?: boolean
```

Comment:
This prop wasn't being used, so I changed it to be a boolean that we can use to toggle the display of the `Search` component